src/jarabe/journal/listview.py

```diff
@@ -711,6 +716,17 @@ def __drag_data_get_cb(self, widget, context, selection, info, time):
         model = self.tree_view.get_model()
         model.do_drag_data_get(path, selection)
 
+    def __button_press_event_cb(self, widget, event):
```

Comment:
Oh, That true! The function would be called even if user is just clicking to start an activity.

I just fixed this . The icon will be set after the `drag-begin` event.